libraries/botbuilder-dialogs-adaptive/tests/LGGenerator.test.js

```diff
@@ -123,6 +123,8 @@ describe('LGLanguageGenerator', function() {
 
         this.beforeAll(async function() {
             const multiLanguageResources = await LanguageResourceLoader.groupByLocale(resourceExplorer);
+
+            //Should have a setup for the threadLocale here.
```

Comment:
See the code in dotnet.
https://github.com/microsoft/botbuilder-dotnet/blob/426b0c306dc25826a5e4837a083e9830213c60eb/tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/LGGeneratorTests.cs#L203